vnext/Desktop/packages.config

```diff
@@ -4,5 +4,5 @@
   <package id="ChakraCore.Debugger" version="0.0.0.42" targetFramework="native" />
   <package id="Microsoft.ChakraCore.vc140" version="1.11.13" targetFramework="native" developmentDependency="true" />
   <package id="Microsoft.Windows.CppWinRT" version="2.0.190730.2" targetFramework="native" />
-  <package id="ReactWindows.OpenSSL.StdCall.Static" version="1.0.2-p.2" targetFramework="native" />
+  <package id="ReactWindows.OpenSSL.v141.Static" version="1.1.1-d.1" targetFramework="native" />
```

Comment:
>v141 [](start = 36, length = 4)

For my understanding, what is v141?

Reply:
`MSVC 14.1`.
Because we may be moving soon to `v142`, I want to keep this build specific and potentially generate a new one using `v142` when the change happens.